while name_3:
    try:
        match name_4:
            case name_3.name_2:
                pass
    except* name_3:
        pass

# output:
# python: Python/flowgraph.c:511: no_redundant_jumps: Assertion `0' failed.
